projects/storefrontlib/src/cms-components/configurator/commons/config-attribute-types/config-attribute-checkbox-list/config-attribute-checkbox-list.component.spec.ts

```diff
@@ -68,4 +62,23 @@ describe('ConfigAttributeDropDownComponent', () => {
     expect(component.attributeCheckBoxForms[1].value).toBe(false);
     expect(component.attributeCheckBoxForms[2].value).toBe(true);
   });
+
+  it('select and deselect a checkbox value', () => {
```

Comment:
Die tests sollten formulieren, was sie erwarten, also eher
it ('should select and deselect a checkbox value')